src/fullnode.rs

```diff
@@ -285,7 +283,7 @@ impl Fullnode {
             cluster_info.clone(),
             config.sigverify_disabled,
             max_tpu_tick_height,
-            blob_index,
```

Comment:
is there a race between when we process the ledger, when the TVU starts up and starts adding blobs to the ledger, and when we get consumed in replay stage?